src/adaptation/Iterator.cc

```diff
@@ -71,7 +71,7 @@ void Adaptation::Iterator::start()
 void Adaptation::Iterator::step()
 {
     ++iterations;
-    debugs(93,5, HERE << '#' << iterations << " plan: " << thePlan);
+    debugs(93,5, '#' << iterations << " plan: " << thePlan);
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.